### PR TITLE
React Front End

### PR DESCRIPTION
**Pull request by jagernet-ops**
# React Front End Pull Request
### Info-for-Herrmann:
Inside of this PR entails all details of the React development. Assign issues accordingly!

### CODE REVIEW

**Comment by jherrmann06** [comment]
Is this ready for review?

**Comment by jagernet-ops** [comment]
> Is this ready for review?

You can merge the current changes. The web app has placeholder values currently.

File: fast-fish/public/manifest.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
+  "short_name": "React App",
```
Comment (jherrmann06) [inline fast-fish/public/manifest.json]: Change to "Fast Fish"

File: fast-fish/public/manifest.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  "name": "Create React App Sample",
```
Comment (jherrmann06) [inline fast-fish/public/manifest.json]: Change to a longer name including "Fast Fish"

**Review by jherrmann06 — CHANGES_REQUESTED** [review]
Mostly looks good, just a tiny change

**Comment by jagernet-ops** [comment]
> Mostly looks good, just a tiny change

I have implemented the change to the manifest file's name, I deleted the old src folder which I glossed over(oops), and I changed the logo images within the public folder to be fishes of different sizes!

**Review by jherrmann06 — APPROVED** [review]
Already did a more informal review in person so I just skimmed this. Looks good

**Pull request merged**